rse-bash.Rmd

```diff
@@ -0,0 +1,3902 @@
+
+> **Manual pages on the web**
+>
+> Of course there is a third way to access help for commands:
+> searching the internet via your web browser.
+> When using internet search, including the phrase `unix man page` in your search
+> query will help to find relevant results.
+>
+> GNU provides links to its
+> [manuals](http://www.gnu.org/manual/manual.html) including the
+> [core GNU utilities](http://www.gnu.org/software/coreutils/manual/coreutils.html),
+> which covers many commands introduced within this lesson.
+
+
+### Looking around and moving about
+
+We can also use `ls` to see the contents of a different directory.  Let's take a
+look at our `Desktop` directory by running `ls -F Desktop`,
+i.e.,
+the command `ls` with the `-F` **option** and the **argument**  `Desktop`.
+The argument `Desktop` tells `ls` that
+we want a listing of something other than our current working directory:
+
+```shell
+$ ls -F Desktop
+```
+
+```text
+data-shell/
+```
+
+Your output should be a list of all the files and sub-directories on your
+Desktop, including the `data-shell` directory you downloaded at
+the [setup for this lesson](http://swcarpentry.github.io/shell-novice/setup.html).
+Take a look at your Desktop to confirm that your output is accurate.
+
+As you may now see, using a bash shell is strongly dependent on the idea that
+your files are organized in a hierarchical file system.
+Organizing things hierarchically in this way helps us keep track of our work:
+it's possible to put hundreds of files in our home directory,
+just as it's possible to pile hundreds of printed papers on our desk,
+but it's a self-defeating strategy.
+
+Now that we know the `data-shell` directory is located on our Desktop, we
+can do two things.
+
+First, we can look at its contents, using the same strategy as before, passing
+a directory name to `ls`:
+
+```shell
+$ ls -F Desktop/data-shell
+```
+
+```text
+creatures/          molecules/          notes.txt           solar.pdf
+data/               north-pacific-gyre/ pizza.cfg           writing/
+```
+
+Second, we can actually change our location to a different directory, so
+we are no longer located in
+our home directory.
+
+The command to change locations is `cd` followed by a
+directory name to change our working directory.
+`cd` stands for "change directory",
+which is a bit misleading:
+the command doesn't change the directory,
+it changes the shell's idea of what directory we are in.
+
+Let's say we want to move to the `data` directory we saw above.  We can
+use the following series of commands to get there:
+
+```shell
+$ cd Desktop
+$ cd data-shell
+$ cd data
+```
+
+These commands will move us from our home directory onto our Desktop, then into
+the `data-shell` directory, then into the `data` directory.  You will notice that `cd` doesn't print anything.  This is normal.  Many shell commands will not output anything to the screen when successfully executed.  But if we run `pwd` after it, we can see that we are now
```

Comment:
please break lines for easier diffing

Reply:
The use of line breaks is a bit inconsistent throughout the document. I've gone through and fixed the most egregious examples.